include/cadmium/engine/pdevs_dynamic_runner.hpp

```diff
@@ -131,6 +138,28 @@ namespace cadmium {
                 void run_until_passivate() {
                     run_until(std::numeric_limits<TIME>::infinity());
                 }
+
+                /**
+                 * @brief Displays current progress of simulation
+                 *  e.g., [50/500]
+                 * 
+                 * @param current Current time step
+                 * @param total Maximum timestep (can be infinity)
+                 */
+                void progress_bar(int current, int total)
```

Comment:
Both `current` and `total` are variables of type `TIME`, not `int`

Reply:
Done